SocialFilter/sections/specific.txt

```diff
@@ -63,6 +63,7 @@ refalover-note.mainichi.jp,note.calbee.jp,note.linemusic.jp,diehardtales.com,not
 !
 speedtest.net.ua###speedo-share
+vip-jikkyo.net###widget-under-sns-buttons
```

Comment:
Please move it to the japanese filter.
![djh](https://user-images.githubusercontent.com/18536328/189640900-39c391dc-cf4f-4af7-9280-47989852d8e8.png)

Reply:
ok.